server/controllers/SolutionController.js

```diff
@@ -128,12 +128,9 @@ exports.edit = async (req, res) => {
   res.status(200).json(updatedSolution);
 };
 
-// TODO: Test this route
+
 exports.delete = async (req, res) => {
   const { id } = req.params;
   await Solution.destroy({ where: { id } });
-  fs.unlink(`${STORAGE_CONFIG.location}/${solution.diagram}`, (err) =>
-    console.log(err),
-  );
```

Comment:
what are these deletions for?

Reply:
thanks for pointing it out I think I forgot about it when I was changing the delete function.